src/platforms/default.rkt

```diff
@@ -6,77 +6,151 @@
+                 #:literal [binary32 32]
+                 [PI.f64 64]
+                 [PI.f32 32]
+                 [E.f64 64]
+                 [E.f32 32]
+                 [INFINITY.f64 64]
+                 [INFINITY.f32 32]
+                 [NAN.f64 64]
+                 [NAN.f32 32]
+                 [neg.f64 128]
+                 [neg.f32 64]
+                 [+.f64 128]
+                 [+.f32 64]
+                 [-.f64 128]
+                 [-.f32 64]
+                 [*.f64 256]
+                 [*.f32 128]
+                 [/.f64 640]
+                 [/.f32 320]
+                 [==.f64 256]
+                 [==.f32 128]
+                 [!=.f64 256]
+                 [!=.f32 128]
+                 [>.f64 256]
+                 [>.f32 128]
+                 [<.f64 256]
+                 [<.f32 128]
+                 [>=.f64 256]
+                 [>=.f32 128]
+                 [<=.f64 256]
+                 [<=.f32 128])
 
 ;; libm operations
-(define libm-platform
-  (with-terminal-cost
-   ([binary64 64] [binary32 32])
-   (let ([rel-costs (cost-map #:default-cost 100 [(fabs) 2] [(sqrt) 10])])
-     (platform-product #:optional [([real binary64]) (cost-map-scale 64 rel-costs)]
-                       [([real binary32]) (cost-map-scale 32 rel-costs)]
-                       (operator-set [(real real)
-                                      (acos acosh
-                                            asin
-                                            asinh
-                                            atan
-                                            atanh
-                                            cbrt
-                                            ceil
-                                            cos
-                                            cosh
-                                            erf
-                                            exp
-                                            exp2
-                                            fabs
-                                            floor
-                                            lgamma
-                                            log
-                                            log10
-                                            log2
-                                            log1p
-                                            logb
-                                            rint
-                                            round
-                                            sin
-                                            sinh
-                                            sqrt
-                                            tan
-                                            tanh
-                                            tgamma
-                                            trunc)]
-                                     [(real real real)
-                                      (atan2 copysign fdim fmax fmin fmod pow remainder)])))))
+(define-platform libm-platform
```

Comment:
You could split this in binary64 and binary32. Wouldn't have to list the cost for most impls